src/__tests__/test-setup.ts

```diff
@@ -0,0 +1,63 @@
+import { MongoMemoryServer } from "mongodb-memory-server"
+import mongoose from "mongoose"
+import flashcardModel from "../models/flashcardModel"
+
+const mongod = MongoMemoryServer.create()
+
+export const startingStateDB = [
```

Comment:
**nitpick:** this const naming is hard to understand, let's change it to `initialCardsMock`